src/main/resources/assets/admin/common/js/ui/treegrid/TreeGrid.ts

```diff
@@ -956,15 +770,21 @@ export class TreeGrid<DATA>
         return this.grid.isAnySelected();
     }
 
+    collapseNodeByRow(row: number) {
+        const nodes: TreeNode<DATA>[] = this.getRoot().getCurrentRoot().treeToList();
+        const draggedNode: TreeNode<DATA> = nodes[row];
```

Comment:
Codacy found an issue: [Variable Assigned to Object Injection Sink](https://app.codacy.com/manual/enonic/lib-admin-ui/pullRequest?prid=6401650)